### ISSUE TITLE
Support addedAt/changedAt/removedAt/movedAt

### ISSUE THREAD

**Issue by veered** [cult-of-coders/redis-oplog#295]
Redis oplog doesn't seem to support the `addedAt`, `changedAt`, `removedAt`, or `movedAt` observe callbacks.

These callbacks are described in the Meteor docs https://docs.meteor.com/api/collections.html#Mongo-Cursor-observe

This could break many use cases, but in particular it breaks Easy Search: https://github.com/matteodem/meteor-easy-search/blob/master/packages/easysearch:core/lib/core/search-collection.js#L217

This can be replicated by running the following code on the server from the Meteor shell:
```js
Meteor.users.find({}).observe({ addedAt(){ console.log('zzz'); } });
```

Note that `zzz` is not printed to the logs when redis oplog is enabled.

**Comment by veered** [comment]
The oplog observe driver falls back to the polling observe driver on ordered callbacks. Perhaps redis oplog should just do this as well.

**Comment by theodorDiaconu** [comment]
It works with the new branch that is in PR, we're only overriding the observerDriver, hooking directly into the core. I still have some small things to do in there, mostly related to sending optimistic ui events async so it doesn't lock.

**Comment by theodorDiaconu** [comment]
@veered yes. The guys from hive.com are using it successfully for over 2 weeks already and they see great improvements, so I would mark it as stable, but I feel that merging it without solving async optimistic ui pushes is like a step-back. I will do my best to fit it in the next weeks to solve it.

**Comment by veered** [comment]
@theodorDiaconu #283 has better performance?

**Comment by theodorDiaconu** [comment]
Havent tested performance. I think that thos is how it should have been coded since the beginning.

Regards,
Theodor

**Comment by etyp** [comment]
@veered I believe this is working now. That PR got merged and released. Did you have a chance to try it out?
